proto/prysm/v1alpha1/attestation.go

```diff
@@ -103,14 +108,36 @@ func (a *Attestation) Version() int {
 	return version.Phase0
 }
 
+// IsNil --
+func (a *Attestation) IsNil() error {
+	if a == nil {
+		return errors.New("attestation is nil")
+	}
+	if a.GetData() == nil {
+		return errors.New("attestation data is nil")
+	}
+	if a.GetData().Source == nil || a.GetData().Target == nil {
+		return errors.New("source/target in attestation data is nil")
+	}
+	if len(a.GetAggregationBits()) == 0 {
+		return errors.New("attestation aggregation bits is empty")
+	}
+
+	if len(a.GetSignature()) == 0 {
```

Comment:
I wonder if we need to check this? The main risk is that code that prepares an attestation for validator signing could run into an error if it uses IsNil somewhere for runtime safety reasons. I feel like these IsNil implementations blur the line between runtime safety and value validation when we start checking for non-zero signature and aggregation bits.

Reply:
it might not be needed, this check was part of the existing REST implementation check, perhaps it's not needed

Reply:
maybe i just need to check outside of this